library.sh

```diff
@@ -752,7 +843,12 @@ function __go_update_deps_for_module() {
 
   if [[ "${FORCE_VENDOR:-false}" == "true" ]] || [ -d vendor ]; then
     group "Go mod vendor"
-    go mod vendor 2>&1 |  grep -v "ignoring symlink" || true
+    # Call go work vendor for Go 1.22+ and go.work file exists.
+    if [ -f "$REPO_ROOT_DIR/go.work" ] && go help work vendor >/dev/null 2>&1; then
```

Comment:
Redirecting all output (stdout and stderr) at once can be done in this way: `go help work vendor &>/dev/null`

Reply:
Nice one. I haven't heard of that!

Reply:
Fixed in https://github.com/knative/hack/pull/386/commits/e922902857fac5afa1a240b25d1c7a25df741685